src/docs/010_manual/50_Frequently_asked_Questions.adoc

```diff
@@ -193,3 +193,24 @@ An easy way to test you configuration is to run `xeyes` in WSL.
 
 NOTE:: Please make sure that you wsl is up-to-date bei executing `wsl --update` - this is not part of your regular windows update!
```

Comment:
Make sure that your WSL is up to date by running `wsl --update`. This is not part of your regular Windows update!